internal/kernel/client.go

```diff
@@ -95,6 +95,7 @@ type RMQClient struct {
 
 	remoteClient *remote.RemotingClient
 	hbMutex      sync.Mutex
+	cancel       context.CancelFunc
```

Comment:
IMO, could we add a field like `state` that is updated to `shutdown` when `Shutdown()` called instead of `cancel`, and return error like `errors.New("client closed")` when `Send*()` called

Reply:
OK，I will adjust the implementation and make it sensible.